### PR TITLE
WSL instructions and heart update.

### PR DESCRIPTION
**Pull request by CrappyDill**
WSL README Instructions.

Added a heart icon with click event to highlight heart.

### CODE REVIEW

File: src/frontend/src/molecules/PostCard.jsx [hunk 1]
```diff
@@ -26,10 +26,21 @@ const PostCard = ({ post }) => {
     }
   }
 
+  function toggleFavorite(e) {
+    e.preventDefault()
+    const heart = e.target
+    console.log(heart.className)
```
Comment (iamdarshshah) [inline src/frontend/src/molecules/PostCard.jsx:32]:
```suggestion
```
Remove console

File: src/frontend/src/molecules/PostCard.jsx [hunk 2]
```diff
@@ -26,10 +26,21 @@ const PostCard = ({ post }) => {
...
+    if (heart.className === "far fa-heart") {
+      heart.className = "fas fa-heart"
+    } else if (heart.className === "fas fa-heart") {
+      heart.className = "far fa-heart"
+    }
```
Comment (iamdarshshah) [inline src/frontend/src/molecules/PostCard.jsx:37]: It'd be great if we include the `else` statement as well to handle the corner case (if any).

File: src/frontend/src/molecules/PostCard.jsx (hunk 1 above)
Comment (CrappyDill) [reply src/frontend/src/molecules/PostCard.jsx:32]: Thank you! Forgot about that.

File: src/frontend/src/molecules/PostCard.jsx (hunk 2 above)
Comment (CrappyDill) [reply src/frontend/src/molecules/PostCard.jsx:37]: Good idea.

File: src/frontend/src/molecules/PostCard.jsx [hunk 3]
```diff
@@ -26,10 +26,20 @@ const PostCard = ({ post }) => {
...
   }
 
+  function toggleFavorite(e) {
+    e.preventDefault()
+    const heart = e.target
+    if (heart.className === "far fa-heart") {
+      heart.className = "fas fa-heart"
+    } else if (heart.className === "fas fa-heart") {
+      heart.className = "far fa-heart"
+    }
```
Comment (monarchwadia) [inline src/frontend/src/molecules/PostCard.jsx:36]: thanks for the contribution @CrappyDill !! great job, this is conceptually great!

quick note for react: in react, application state drives the DOM.  one never modifies the DOM directly. we actually want to use the `useState` hook in this case.

I'll accept this in and then refactor it, then share the new code with you :-)

Comment (monarchwadia) [reply src/frontend/src/molecules/PostCard.jsx:36]: Here you go: https://discord.com/channels/@me/846852959886901308/849697494761078795

**Pull request merged**